docs/en/simulation/hitl.md

```diff
@@ -101,16 +101,17 @@ make px4_fmu-v6x boardconfig
    2. Select a [compatible airframe](#compatible_airframe) you want to test.
       Then click **Apply and Restart** on top-right of the _Airframe Setup_ page.
 
-3. Calibrate your RC or Joystick, if needed.
+3. Calibrate your [Manual Controller](../config/manual_control.md) (RC or Joystick), if needed.
 4. Setup UDP
    1. Under the _General_ tab of the settings menu, uncheck all _AutoConnect_ boxes except for **UDP**.
 
       ![QGC Auto-connect settings for HITL](../../assets/gcs/qgc_hitl_autoconnect.png)
 
 5. (Optional) Configure Joystick and Failsafe.
-   Set the following [parameters](../advanced_config/parameters.md) in order to use a joystick instead of an RC remote control transmitter:
-   - [COM_RC_IN_MODE](../advanced_config/parameter_reference.md#COM_RC_IN_MODE) to "Joystick/No RC Checks". This allows joystick input and disables RC input checks.
-   - [NAV_RCL_ACT](../advanced_config/parameter_reference.md#NAV_RCL_ACT) to "Disabled". This ensures that no RC failsafe actions interfere when not running HITL with a radio control.
+   - [Enable an COM_RC_IN_MODE mode that allows Joystick](../config/manual_control.md#px4-configuration).
+     The default `RC or MAVLink keep first` should work if you plan to only have a Joystick (no RC).
+   - Set [NAV_RCL_ACT](../advanced_config/parameter_reference.md#NAV_RCL_ACT) to `Disabled`.
+     This ensures that no RC failsafe actions interfere when not running HITL with a radio control.
```

Comment:
I'm wondering if for this case we should make it purely generic "configure your priority" rather than "configure your joystick". Then at the end we'd just have "If you're using a Joystick also set NAV_RCL_ACT ..."